themes/default/content/docs/guides/self-hosted/ecs-hosted.md

```diff
@@ -0,0 +1,79 @@
+---
+title: ECS-Hosted Install
+menu:
+    userguides:
+        parent: self_hosted
+        identifier: self_hosted_ecs_hosted_install
+        weight: 20
+meta_desc: Installer for deploying the self-hosted Pulumi service in ECS.
+---
+
+The [ECS-Hosted Installer](https://github.com/pulumi/pulumi-self-hosted-installers/ecs-hosted) is used to deploy the self-hosted Pulumi service in Amazon Elastic Container Service.  
+
+## Prerequisites
+
+The customer is required to provide and manage the following:
+
+* AWS VPC with
+  * At least 2 public subnets available.
+  * At least 2 private subnets available.
+  * At least 2 isolated subnets available.
+    * An "isolated" subnet means it can only route traffic within the subnet. So there is no NAT gateway.
+* Route53 hosted zone.
+* ACM Certiciate that covers FQDNs of the form (note `{subdomain}` is optional):
```

Comment:
```suggestion
* ACM Certificate that covers FQDNs of the following form, where `{subdomain}` is optional:
```

It might also be good to show what these look like when there isn't a subdomain, just so the reader doesn't have to guess at it, but we could come back and do that later, too.

Reply:
made the typographical change. But I couldn't come up with a pretty way to show the no-subdomain case. So I'm leaving it as is for now.